data/README.md

```diff
@@ -0,0 +1,40 @@
+# Datasets (This section is work in progress.)
+
+## Anomalies datasets
+
+There are several types of anomalies in data streams. We categorize them in the following way:
+- neatData – clean data sets generated with no anomalies included. Eg. sine, constant, spike train.
+- corruptedData – these data sets has been taken from the previous group but then corrupted by deterministic anomalies. Subfolders comprise datasets grouped by the category. Such as: point anomaly, section anomaly, etc. 
```

Comment:
Maybe "Subfolders include specific anomaly types (categories) which we have identified, such as: point anomaly, section anomaly, etc. " ?